examples/routing-v1/client/main.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"time"
+
+	"github.com/ipfs/boxo/ipns"
+	"github.com/ipfs/boxo/routing/http/client"
+	"github.com/ipfs/boxo/routing/http/types"
+	"github.com/ipfs/boxo/routing/http/types/iter"
+	"github.com/ipfs/go-cid"
+	"github.com/libp2p/go-libp2p/core/peer"
+)
+
+func main() {
+	gatewayUrlPtr := flag.String("e", "", "routing v1 endpoint to use")
+	timeoutPtr := flag.Int("t", 10, "timeout in seconds for lookup")
+	cidPtr := flag.String("c", "", "cid to find")
+	pidPtr := flag.String("p", "", "peer to find")
+	namePtr := flag.String("n", "", "ipns name to retrieve record for")
```

Comment:
:thought_balloon:  since this is an example, using meaningful parameters makes one-liners on slides and blogs easier to parse and understand:
```suggestion
	cidPtr := flag.String("cid", "", "cid to find")
	pidPtr := flag.String("peer", "", "peer to find")
	namePtr := flag.String("ipns", "", "ipns name to retrieve record for")
```